Makefile

```diff
@@ -65,10 +65,19 @@ lint-with-docker:  ## Lint source code with docker image of golangci-lint
 .PHONY: lint lint-with-local lint-with-docker
 
 # --- Docker -------------------------------------------------------------------
+TAG ?= $(error tag not set)
 
 docker-build:
 	docker build --tag foxtrot:latest .
 
+docker-build-release:
+	docker buildx build \
+		--push \
+		--tag foxygoat/foxtrot:latest \
+		--tag foxygoat/foxtrot:$(TAG) \
+		--platform linux/amd64,linux/arm/v7 .
+
+
```

Comment:
nit: extra unnecessary blank line.

Perhaps put `.PHONY: docker-build docker-build-release` here to make it non-blank?

Reply:
done. eagle eyes as always. thank you